web/src/features/panels/zone/MethodologyCard.tsx

```diff
@@ -7,16 +7,16 @@ import { useState } from 'react';
 import trackEvent from 'utils/analytics';
 
 export default function MethodologyCard() {
-  const [isCollapsed, setIsCollapsed] = useState(true);
+  const [accordionState, setAccordionState] = useState(true);
```

Comment:
Consider using`isCollapsed/setIsCollapsed`.  `accordionState === true` is kind of confusing semantically